mgtk.inc

```diff
@@ -607,6 +607,24 @@ winfo_offset_port               := 20
 winfo_offset_nextwinfo          := 56
 winfo_size                      := 58
 
+
+;;; Errors
+
+error_in_object := $80
```

Comment:
There is already `inrect_inside` and `inpoly_inside` (and `*_outside`) defined above, which are used by desktop.s and some overlays (apparently I didn't update the DAs to use them, though). 

We should de-dupe. I don't have a strong preference on how, though.

Reply:
Ah, yes... I didn't notice that. I'll make them the same somehow.